src/web/js/trove/chart-lib.js

```diff
@@ -268,23 +418,259 @@
+      colors_list = colors_list.slice(0, legends.length);
+    }
+
+    // Initializes the Columns of the data 
     data.addColumn('string', 'Label');
     legends.forEach(legend => data.addColumn('number', legend));
+
+    // Adds each row of bar data
     data.addRows(table.map(row => [row[0]].concat(row[1].map(n => toFixnum(n)))));
+    addAnnotations(data, rawData);
+    addIntervals(data, rawData);
+
+    let options = {
+      isStacked: get(rawData, 'is-stacked'),
+      series: colors_list.map(c => ({color: c, targetAxisIndex: 0})),
+      legend: {
+        position: horizontal ? 'right' : 'top', 
+        maxLines: data.getNumberOfColumns() - 1
+      }, 
+      intervals: { 
+        color : interval_color, 
+      }
+    };
+
+    
+    options[axisloc] = { 
+      0: { 
+        viewWindow: { max: axis.top, min: axis.bottom }, 
+        ticks: axis.ticks 
+      }
+    };
+         
+    /* NOTE(John & Edward, Dec 2020): 
+       Our goal for the part below was to add pointers (Specific Named Ticks) on another VAxis. 
+       The Current Chart library necessitates that we assign at least one stack/bar to the 
+       second axis in order for it to show up, and we have to fix the min/max of each axis 
+       manually to make sure that both are consistent with each other rather than being relative 
+       to the data. There is also a problem: When the pointers are too close to each other, one or 
+       both of them disappear!
+    */
+
+    if (pointers_list.length > 0) { 
+      colors_list = colors_list.slice(0, legends.length);
+      
+      // Add and Attach Empty Data Stack/bar to 2nd axis + Color it
+      data.addColumn('number', 'Pointers')
+
+      for (let i = 0; i < data.getNumberOfColumns() - 1; i++) {
+        if (options['series'][i] == null) {
+          options['series'][i] = {color: pointer_color, targetAxisIndex: 1};
+        }
+      }
+
+      // Update Options to include the new axis ticks consistent with the first axis
+      options[axisloc][1] = { 
+        viewWindow: { 
+          max: axis.top, 
+          min: axis.bottom
+        },
+        gridlines: { color: pointer_color },
+        ticks: pointers_list, 
+        textStyle: { color: pointer_color }
+      };
+    } else {
+      for (let i = 0; i < data.getNumberOfColumns() - 1; i++) {
+        if (options['series'][i] == null) {
+          options['series'][i] = {color: 'black', targetAxisIndex: 0};
+        }
+      }
+    }
+      
     return {
       data: data,
-      options: {
-        legend: {
-          position: isTrue(get(rawData, 'has-legend')) ? 'right' : 'none'
-        }
-      },
-      chartType: google.visualization.ColumnChart,
+      options: options,
+      chartType: horizontal ? google.visualization.BarChart : google.visualization.ColumnChart,
       onExit: defaultImageReturn,
-      mutators: [axesNameMutator, yAxisRangeMutator],
+      mutatorgraphs: [axesNameMutator, yAxisRangeMutator],
```

Comment:
Why is this key changed? Don't see `mutatorgraphs` mentioned elsewhere in the file.

Reply:
I'm going to be 100% honest. I have no idea why, I think this might have been a change made in one of the meetings we had with @blerner since I don't recall really ever changing this on my own, and I don't really remember what it does. I would check in with ben!

Reply:
@blerner can you check this? It looks incorrect to me.

Here's a program that demonstrates the bug:

```

include chart

a-stacked-bar-chart-series = from-list.stacked-bar-chart(
  [list: 'CA', 'TX', 'NY', 'FL', 'IL', 'PA'],
  [list:
    [list: 2704659,4499890,2159981,3853788,10604510,8819342,4114496],
    [list: 2027307,3277946,1420518,2454721,7017731,5656528,2472223],
    [list: 1208495,2141490,1058031,1999120,5355235,5120254,2607672],
    [list: 1140516,1938695,925060,1607297,4782119,4746856,3187797],
    [list: 894368,1558919,725973,1311479,3596343,3239173,1575308],
    [list: 737462,1345341,679201,1203944,3157759,3414001,1910571]],
  [list:
    'Under 5 Years',
    '5 to 13 Years',
    '14 to 17 Years',
    '18 to 24 Years',
    '25 to 44 Years',
    '45 to 64 Years',
    '65 Years and Over'])


render-chart(a-stacked-bar-chart-series)
  .x-axis("hello")
  .display()
```

Reply:
Joe and I saw that, and know it's wrong -- the question is whether it should be renamed back to `mutators`, or deleted entirely.  @JohnUUU needs to check it and see which behavior is intended.

Reply:
Yeah definitely, I'll do that right now!

Reply:
Oh I'm just realizing that new commits no longer get forwarded here since the PR has been merged. Should I open another PR? If you want to just make the change directly on horizon the right behavior happens when we rename it back to mutators.